dump/src/main.rs

```diff
@@ -37,6 +44,14 @@ struct App {
     /// Fail if any errors are encountered
     #[clap(long = "fail-first")]
     fail_first: bool,
+    /// Output format
+    #[arg(value_enum)]
+    #[clap(short = 'o', long = "output", default_value = "text")]
```

Comment:
I'd say we rename the option to `format` (short `-F`, long `--format`), which not only aligns more with the library, but also does not give the false impression that this is for an output file path, with `-o` often being used for this purpose.

Reply:
Sounds good and makes sense, I am just used to `-o yaml` and `-o json` from `kubectl`!